packages/twenty-front/src/modules/workflow/workflow-variables/components/WorkflowTextEditorVariableChip.tsx

```diff
@@ -12,14 +12,18 @@ type WorkflowTextEditorVariableChipProps = NodeViewProps;
 export const WorkflowTextEditorVariableChip = ({
   deleteNode,
   node,
+  editor,
 }: WorkflowTextEditorVariableChipProps) => {
   const attrs = node.attrs as {
     variable: string;
   };
 
   return (
     <NodeViewWrapper as={StyledWrapper} style={{ whiteSpace: 'nowrap' }}>
-      <VariableChip rawVariableName={attrs.variable} onRemove={deleteNode} />
+      <VariableChip
+        rawVariableName={attrs.variable}
+        onRemove={editor.isEditable ? deleteNode : undefined}
```

Comment:
Could be made more explicit to read in a handleRemoveClick with a if(editor.isEditable) { deleteNode() }

Reply:
If the `onRemove` property is not provided or is `undefined`, we hide an inner button to delete the variable chip. I'm unsure of a better way to do this.
We could provide a `disabled` property, but I don't know if it would clarify things.

Reply:
Yes most probably, a readonly prop would make this more explicit and avoid guessing this behavior, indeed.